mmcv/runner/fp16_utils.py

```diff
@@ -21,7 +22,10 @@
     pass
 
 
-def cast_tensor_type(inputs, src_type, dst_type):
+def cast_tensor_type(inputs: Union[torch.Tensor, nn.Module, str, np.ndarray,
+                                   abc.Mapping, abc.Iterable],
+                     src_type: torch.dtype,
+                     dst_type: torch.dtype) -> torch.Tensor:
```

Comment:
The return type should be `Union[torch.Tensor, nn.Module, str, 
                                   abc.Mapping, abc.Iterable],`